### DISCUSSION TITLE
How to more easily add handlers to the <body> in Nuxt 3

### DISCUSSION THREAD

**Discussion by damianof** [nuxt/nuxt#24839] [Questions]
I need to add `ontouchstart=""` to the <body> element and the Nuxt documentation only talks about how to add attributes to the <body> via useHead or nuxt.config.js

It would be nice if the template for the index.html page was accessible more easily like in Vue

**Comment by TheAlexLichter** [comment]
You can hook into the `render:html` nitro hook to manipulate HTML on the server if you need it for every page.
This can be done via a Nitro plugin.

**Comment by arashsheyda** [comment]
what about using `<body>` tag in `app.vue`?

```app.vue
<Body ontouchstart="">
	<NuxtPage />
</Body>
```

**Comment by harlan-zw** [comment]
Seems to work fine using `useHead` https://stackblitz.com/edit/nuxt-starter-hjuap7?file=app.vue

```ts
<script lang="ts" setup>
useServerHead({
  bodyAttrs: {
    ontouchstart: '',
  },
});
</script>
```

This is equivalent to what @arashsheyda has said, except `<Body>` will use `useHead`. I'd use useServerHead if you don't want to fire any specific code, this way it's not bundled on the client side.

You can even do it from your nuxt config for extra no-bundling.

If you need the `''` you'll need to provide some value, I'd recommend `void(0)` if you just want to return `undefined`.

Documentation is here https://unhead.unjs.io/usage/guides/dom-event-handling, could use some improvements.
